src/main/java/uk/gov/ons/ctp/response/sample/service/SampleService.java

```diff
@@ -200,6 +200,26 @@ public Integer initialiseCollectionExerciseJob(CollectionExerciseJob job) throws
     return sampleUnitsTotal;
   }
 
+  public int getSampleSummaryUnitCount(UUID sampleSummaryId) {
+    int sampleUnitsTotal = 0;
+
+    SampleSummary sampleSummary = sampleSummaryRepository.findById(sampleSummaryId);
+
+    if (sampleSummary != null) {
```

Comment:
Can we use guard clauses instead of a nested if statement here? 
``` 
if (sampleSummary == null) {
throw new IllegalArgumentException...
}

if(sampleSummary.getTotalSampleUnits() == null) {
Throw new IllegalStateException...
}

return sampleSummary.getTotalSampleUnits().intValue();
```

Reply:
Yes. Will do